config/samples/assistedservice.openshift.io_v1alpha1_assistedservice.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: assistedservice.openshift.io.openshift.io/v1alpha1
+kind: AssistedService
+metadata:
+  name: assistedservice-sample
+spec:
+  # Add fields here
+  foo: bar
```

Comment:
Should this be a real example?

Reply:
No :). Will remove because the AssistedService CRD doesn't exist.